### ISSUE TITLE
no-standalone-expect reports false positive when "test" is aliased

### ISSUE THREAD

**Issue by msorens** [playwright-community/eslint-plugin-playwright#340]
Simply aliasing `test` to, for example, `setup`, causes an eslint violation "Expect must be inside of a test block". Any reason this should not be considered valid?

Minimal test case:
```
import { expect, test as setup } from '#test'
. . .
setup('authenticate', async ({ page, system }) => {
    expect(true).toBeTruthy()
})

**Comment by mskelton** [comment]
Definitely something we should support. In the meantime, you can use `globalAliases` https://github.com/playwright-community/eslint-plugin-playwright#settings

**Comment by msorens** [comment]
What file should contain the globalAliases (and other) settings? Your doc does not specify.

**Comment by mskelton** [comment]
https://eslint.org/docs/latest/use/configure/configuration-files#configuring-shared-settings
